extension/intents/search/search.js

```diff
@@ -244,6 +244,56 @@ this.intents.search = (function() {
     },
   });
 
+  intentRunner.registerIntent({
+    name: "search.previous",
+    description:
+      "If you've done a search then this will display the previous search result.",
+    examples: ["test:previous search item"],
+    match: `
+    (search |) previous (search |) (result{s} | item{s} | page | article |)
+    `,
+    async run(context) {
```

Comment:
Probably `.run()` should call something like `moveResult(-1)` (and then "next" would call `moveResult(1)`) – since most of this code is the same

Reply:
Thank you for the feedback. After my holidays I,ll continue working on it.